app/src/main/java/com/sun/weather04/data/repository/WeatherRepository.java

```diff
@@ -0,0 +1,26 @@
+package com.sun.weather04.data.repository;
+
+import com.sun.weather04.data.model.DataResponse;
+import com.sun.weather04.data.source.WeatherDataSource;
+import com.sun.weather04.data.source.remote.OnFetchDataListener;
+
+public class WeatherRepository {
+    private static WeatherRepository sInstance;
+    private WeatherDataSource.RemoteDataSource mRemoteDataSource;
+
+    private WeatherRepository(WeatherDataSource.RemoteDataSource remoteDataSource) {
+        mRemoteDataSource = remoteDataSource;
+    }
+
+    public static WeatherRepository getInstance(
+            WeatherDataSource.RemoteDataSource remoteDataSource) {
```

Comment:
```suggestion
            public static WeatherRepository getInstance(WeatherDataSource.RemoteDataSource remoteDataSource) {
```
Cho nó cùng một dòng luôn